org.nomacs.ImageLounge.json

```diff
@@ -10,7 +10,8 @@
         "--socket=wayland",
         "--socket=pulseaudio",
         "--filesystem=home",
-        "--filesystem=/media"
+        "--filesystem=/media",
+        "--filesystem=xdg-run/gvfs"
```

Comment:
don't you need the dbus talk-name too for gvfs ?

Reply:
Sorry I'm not sure what do you mean. Giving Nomacs permissions to access "xdg-run/gvfs" should be enough, or not?
For me it works, and I'm able to browse pictures on network shares.
I'm no expert on this matter, what should "dbus talk-name" do?